src/container/Calendar.react.js

```diff
@@ -5,16 +5,17 @@
 
```

Comment:
it won't let me comment on the actual line, but I think if you remove `@flow` from the top of this file (and any others) that might fix ya up too.

Reply:
Not sure it would. If `@flow` isn't specified, the `type` keywords would likely fail. Also don't think props supports passing a class as an valid prop type.